docs/examples/namespace-offloading-default.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: offloading.liqo.io/v1alpha1
+kind: NamespaceOffloading
+metadata:
+  name: offloading
+  namespace: your-namespace
```

```suggestion
  namespace: target-namespace
```

Just for consistency